index.bs

```diff
@@ -4534,7 +4534,7 @@ the attestation=] is consistent with the fields of the attestation certificate's
 ### Android Key Attestation Statement Certificate Requirements ### {#sctn-key-attstn-cert-requirements}
 
 Android Key Attestation [=attestation certificate=]'s <dfn>android key attestation certificate extension
-data</dfn> is identified by the OID "1.3.6.1.4.1.11129.2.1.17".
+data</dfn> is identified by the OID "1.3.6.1.4.1.11129.2.1.17", and its [schema is defined here](https://developer.android.com/training/articles/security-key-attestation#certificate_schema).
```

Comment:
Earlier in this section, and the next section, we use the style "See the [Android developer documentation]" / "See [SafetyNet Documentation] for more details", so for consistency:

```suggestion
data</dfn> is identified by the OID "1.3.6.1.4.1.11129.2.1.17", and its schema is defined in the [Android developer documentation](https://developer.android.com/training/articles/security-key-attestation#certificate_schema).
```

Reply:
thanks!